File: TaskManager/TrelloApp/TrelloApp/Comment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TrelloApp
{
    public class Comment
    {
        public int id;
        public string content;
        public string author;
        public int createdAt;

        public Comment (int id, string content, string author, int createdAt)
        {
            this.id = id;
            this.content = content;
            this.author = author;
            this.createdAt = createdAt;
        }
    }
}

File: TaskManager/src/Comment.java
import java.util.Date;

public class Comment {
    public int ID;
    public String Content;
    public String Author;
    public Date CreatedAt;
    public Comment(int id, String content, String Author, Date CreatedAt)
    {
        this.ID = id;
        this.Content = content;
        this.Author = Author;
        this.CreatedAt = CreatedAt;
    }
}
